cli/cmd/uninstall.go

```diff
@@ -143,6 +151,25 @@ func UninstallClusterResources(ctx context.Context, client *kube.Client, ns stri
 
 }
 
+func waitForPodsToRolloutWithoutInstrumentation(ctx context.Context, client *kube.Client) {
+	l := log.Print("Rolling back odigos changes to pods...")
+	instrumentedPodReq, _ := k8slabels.NewRequirement(k8sconsts.OdigosAgentsMetaHashLabel, selection.Exists, []string{})
+	wait.PollImmediate(1*time.Second, 5*time.Minute, func() (bool, error) {
+		pods, err := client.CoreV1().Pods("").List(ctx, metav1.ListOptions{
+			Limit: 1,
+			LabelSelector: instrumentedPodReq.String(),
+		})
+		if err != nil {
+			return false, err
+		}
+		if len(pods.Items) == 0 {
+			l.Success()
+			return true, nil
+		}
+		return false, nil
+	})
+}
```

Comment:
suggestion: print if all pods are rolled-out successfully or if we timed out and implications